func_programming/3.py

```diff
@@ -0,0 +1,17 @@
+original_list = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10]
+
+square_function = lambda x: x ** 2
+
+cube_function = lambda x: x ** 3
+
+squares = list(map(square_function, original_list))
+cubes = list(map(cube_function, original_list))
```

Comment:
można też bez nazywanie lambd, można z nawzami :)
```suggestion
squares = list(map(lambda x: x ** 2, original_list))
cubes = list(map(lambda x: x ** 3, original_list))
```